components/resources/BUILD.gn

```diff
@@ -89,11 +89,16 @@ repack("resources") {
     ]
   }
 
-  if (brave_wallet_enabled && !is_android) {
-    deps += [ "//brave/components/brave_wallet_ui:resources" ]
-    sources += [
-      "$root_gen_dir/brave/components/brave_wallet/resources/brave_wallet.pak",
-    ]
+  if (brave_wallet_enabled) {
+    if (!is_android) {
+      deps += [ "//brave/components/brave_wallet_ui:resources" ]
+      sources += [
+        "$root_gen_dir/brave/components/brave_wallet/resources/brave_wallet.pak",
```

Comment:
shouldn't this be `brave_wallet_script.pak` as that's what the `resource_name is` above.

Reply:
oh n/m I was confused by the diff and thought this was new code. Ignore.